std/assembly/internal/typedarray.ts

```diff
@@ -63,8 +63,28 @@ export abstract class TypedArray<T,V> {
 
   // copyWithin(target: i32, start: i32, end: i32 = this.length): this
 
+  fill(value: V, start: i32 = 0, end: i32 = i32.MAX_VALUE): this {
+    var buffer = this.buffer;
+    var byteOffset = this.byteOffset;
+    var len = this.length;
+    start = start < 0 ? max(len + start, byteOffset >> alignof<T>()) : min(start, len);
```

Comment:
Not sure about lower and upper limits. Need more tests

Reply:
Should be possible to ignore byteOffset up to the point where the `memory.fill` operation is used, like with a normal array. That's just the offset into the backing buffer when accessing it internally.